base/sparse/cholmod.jl

```diff
@@ -1379,9 +1385,11 @@ function ldltfact(A::Sparse; shift::Real=0.0,
 end
 
 """
-    ldltfact(::Union{SparseMatrixCSC{<:Real},SparseMatrixCSC{Complex{<:Real}},Symmetric{<:Real,SparseMatrixCSC{<:Real,SuiteSparse_long}},Hermitian{Complex{<:Real},SparseMatrixCSC{Complex{<:Real},SuiteSparse_long}}}; shift = 0.0, perm=Int[]) -> CHOLMOD.Factor
+    ldltfact(A; shift = 0.0, perm=Int[]) -> CHOLMOD.Factor
 
-Compute the ``LDL'`` factorization of a sparse symmetric or Hermitian matrix.
+Compute the ``LDL'`` factorization of a sparse matrix `A`.
```

Comment:
You're right, this won't work. I was trying to find a way to say you don't have to wrap the sparse matrix in a tag like `Symmetric`.